tsl/test/sql/exp_cagg_monthly.sql

```diff
@@ -66,10 +66,6 @@ FROM _timescaledb_catalog.continuous_agg
 WHERE user_view_name = 'conditions_summary'
 \gset
 
-SELECT bucket_width
-FROM _timescaledb_catalog.continuous_agg
-WHERE mat_hypertable_id = :cagg_id;
-
```

Comment:
Why remove it and not expand the query JOINING with the continuous_agg_bucket_function to show the bucket width?

Same applies to other two queries removed from exp_cagg_timezone.sql

Reply:
This is tested in the next line. In this line, `continuous_aggs_bucket_function` is queried for the `cagg_id`. The same is true for the removed lines in `exp_cagg_timezone`, `continuous_aggs_bucket_function.bucket_width` is read in the next lines.